tests/lib/test-common.bash.in

```diff
@@ -39,6 +39,17 @@ die(){
 	echo "ERROR: $msg" >&2
 	exit 1
 }
+#used it to fail a bats test and show an error message
+fail (){
+	msg="$*"
+	echo "FAIL: $msg" >&2
+	false
+}
+
+info (){
+	msg="$*"
+	echo "INFO: $msg" >&2
```

Comment:
I prefer to send info to stderr just un case someone make a function and want to use the output for the command they did , but without having issues with info log. But I can change it if that does not sounds like a good approach.